ci/jenkins/test.sh

```diff
@@ -69,7 +70,8 @@ Run K8s e2e community tests (Conformance & Network Policy) or Antrea e2e tests o
         --testbed-type           The testbed type to run tests. It can be flexible-ipam, jumper or legacy.
         --ip-mode                IP mode for flexible-ipam e2e test. Default is $DEFAULT_IP_MODE. It can also be ipv6 or ds.
         --win-image-node         Name of the windows image node in containerd cluster. Images are built by docker on this node.
-        --kind-cluster-name      Name of the kind Cluster." 
+        --kind-cluster-name      Name of the kind Cluster.
+        --build-tag              Custom build tag for images." 
```

Comment:
Please clarify in the comments that the `build-tag` is now only available for kind tests. 
Additionally, the script should return an error message if the `build-tag` is not provided when `testbed-type` is 'kind', or if it is provided when `testbed-type` is not 'kind'.

Reply:
but we should not return an error when build-tag is not provided and testbed-type is kind, because it is not necessary, it is only required when we want to run multiple jobs on the same testbed., similarly we should not return an error, when build-tag is provided and testbed-type is not kind, because it won't affect anything else in the code, it just gives the user a feature to provide their own tag if they want for some testing purpose.

Reply:
> but we should not return an error when build-tag is not provided and testbed-type is kind, because it is not necessary, it is only required when we want to run multiple jobs on the same testbed., similarly we should not return an error, when build-tag is provided and testbed-type is not kind, because it won't affect anything else in the code, it just gives the user a feature to provide their own tag if they want for some testing purpose.

If build-tag is not provided in test.sh, in Makefile a new tag will be generated from build-tag.sh, which will differ from the CUSTOM_BUILD_TAG default value. Then how can the `kind load` command correctly load the image?

Reply:
if no build-tag is  provided then the CUSTOM_BUILD_TAG will have its value set to the BUILD_TAG generated by build-tag.sh script.

Reply:
May I know where you set CUSTOM_BUILD_TAG the value generated by build-tag.sh in test.sh? I understand the image will have new tag generated by build-tag.sh, but in line607 `CUSTOM_BUILD_TAG` should still be the default value "", is that correct?

Reply:
yes that case, it will be the latest tag, which is the default tag, in use, may be in test.sh i can set the default value of the tag as latest.